api/api/test/test_util.py

```diff
@@ -302,3 +302,16 @@ def dummy_func():
         assert response.headers.pop('Link') == f'<{link}>; rel="Deprecated"', 'No link was found'
 
     assert response.headers == {}, f'Unexpected deprecation headers were found: {response.headers}'
+
+
+@patch('api.util.raise_if_exc')
+def test_only_master_endpoint(mock_exc):

```

Comment:
Test docstring missing. Also, should we add a validation that `func_` was executed properly, or do we take it for granted since the test passes?